apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -25,48 +45,28 @@ export default function SchoolDataInputs({
     schoolZip: 'user[school_info_attributes][school_zip]',
   },
 }) {
-  // If the user filled out country before or we are detecting a US IP address
-  const detectedCountry =
-    sessionStorage.getItem(SCHOOL_COUNTRY) || (usIp ? US_COUNTRY_CODE : '');
-  const [country, setCountry] = useState(detectedCountry);
-  const [askForZip, setAskForZip] = useState(
-    detectedCountry === US_COUNTRY_CODE
-  );
-  const [isOutsideUS, setIsOutsideUS] = useState(
-    (detectedCountry || usIp === false) && detectedCountry !== US_COUNTRY_CODE
-  );
+  // We don't want to display any fields to start that won't eventually be
+  // necessary, so updating any time country changes
+  const countryIsUS = useMemo(() => country === US_COUNTRY_CODE, [country]);
+
+  const inputManually = useMemo(() => schoolId === CLICK_TO_ADD, [schoolId]);
 
-  // Add 'Select a country' and 'United States' to the top of the country list
-  let COUNTRY_ITEMS = [
-    {value: 'selectCountry', text: i18n.selectCountry()},
-    {value: US_COUNTRY_CODE, text: i18n.unitedStates()},
-  ];
-  // Pull in the rest of the countries after/below
-  const nonUsCountries = Object.values(COUNTRIES).filter(
-    item => item.label !== US_COUNTRY_CODE
+  const schoolSelectOptions = useMemo(
+    () => [
+      {value: SELECT_A_SCHOOL, text: i18n.selectASchool()},
+      ...schoolsList,
+    ],
+    [schoolsList]
   );
-  for (const item of nonUsCountries) {
-    COUNTRY_ITEMS.push({value: item.label, text: item.value});
-  }
 
-  const onCountryChange = e => {
-    const country = e.target.value;
-    setCountry(country);
-    sessionStorage.setItem(SCHOOL_COUNTRY, country);
-    analyticsReporter.sendEvent(
-      EVENTS.COUNTRY_SELECTED,
-      {country: country},
-      PLATFORMS.BOTH
-    );
-    // We don't want to display any fields to start that won't eventually be
-    // necessary, so updating both of these any time country changes
-    if (country === US_COUNTRY_CODE) {
-      setAskForZip(true);
-      setIsOutsideUS(false);
-    } else {
-      setAskForZip(false);
-      setIsOutsideUS(true);
-    }
+  const handleCountryChange = c => {
+    setCountry(c);
+  };
```

Comment:
Nit: could the `onChange` just be `e => setCountry(e)` below rather than having this `handleCountryChange`?